.github/workflows/README.md

```diff
@@ -0,0 +1,152 @@
+# AlphaX Release Process
+These [reusable workflows](https://docs.github.com/en/actions/sharing-automations/reusing-workflows) aim at unifying the release process for all AlphaX packages.
+They automate the creation of a draft release on GitHub, the building of installers for different platforms,
+testing of the installers, the publishing of the package on PyPi and (optionally) the publishing on Docker Hub.
+
+The first part of this Readme explains the release process itself.
+The second part shows how to incorporate this pipeline into your repository.
+The last part explains how to further develop this pipeline.
+
+## Release a new version
+
+### Prerequisites
+Note: these instructions assume that the release is done from the `main` branch,
+and that the following worklows are present in the repository (cf. [below](#installation-of-the-release-pipeline)):
+- 'Create Draft Release' (referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/create_release.yml))
+- 'Publish on PyPi' (referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/publish_on_pypi.yml))
+- 'Publish Docker Image' (referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/publish_docker_image.yml))
+
+
+### Step-by-step instructions
```

Comment:
what is the best way to do test releases with this pipeline? For example, should we do a separate branch and use bumpversion `9.9.9_testversion`?

Reply:
good question, added this to the Readme: https://github.com/MannLabs/alphashared/pull/4/commits/acc9f0735e017cd0904fff66812555bf2cb9cd3b